src/server/items/validator.js

```diff
@@ -10,21 +10,20 @@ export default (req, res, next) => {
   const MAX_LNG = 180;
   const CATEGORIES = ['warning', 'event', 'facility'];
   const key = req.params.id;
-  if (key && !(key.startsWith('item-')
-      && validator.isUUID(key.substr(5, key.length)))) {
+  if (key && !(key.startsWith('item-'))) {
     return next(new APIError(new Error(), {
       statusCode: 400,
       message: 'wrong item Id'
     }));
   }
-  if (req.body.description && !(typeof req.body.description === 'string'
-    && validator.isLength(req.body.description, {
+  if (req.body.title && !(typeof req.body.title === 'string'
+    && validator.isLength(req.body.title, {
       min: MIN_DECSRIPSION_LENGTH,
       max: MAX_DECSRIPSION_LENGTH
```

Comment:
Lets edit these const vars to 'MIN_TITLE_LENGTH' too.
And we have keyMaker module now, so we can refer const vars about making key to key Maker Module.